project_forecast_line/models/hr_employee.py

```diff
@@ -15,9 +15,16 @@ class HrJob(models.Model):
 class HrEmployee(models.Model):
     _inherit = "hr.employee"
 
-    role_ids = fields.One2many("hr.employee.forecast.role", "employee_id")
+    role_ids = fields.One2many(
```

Comment:
thanks @thomaspaulb for the fix. Would be great if you can simply add a comment before these fields to explain why we need `groups="hr.group_hr_user`
Some link to the https://github.com/odoo/odoo/blob/d21887008b1ac769bd91d24e972323ffe936391a/addons/hr/models/hr_employee.py#L22

Reply:
Done